pkg/csi/cinder/openstack/openstack_snapshots.go

```diff
@@ -51,7 +62,7 @@ func (os *OpenStack) CreateSnapshot(name, volID, description string, tags *map[s
 // operation.  Valid filter keys are:  Name, Status, VolumeID (TenantID has no effect)
 func (os *OpenStack) ListSnapshots(limit, offset int, filters map[string]string) ([]snapshots.Snapshot, error) {
 	// FIXME: honor the limit, offset and filters later
-	opts := snapshots.ListOpts{}
+	opts := snapshots.ListOpts{Status: SnapshotReadyStatus}
```

Comment:
if we want have some error in creating the snapshot, will it be stuck somewhere and we can't even find them later on given the condidtion is ready only?

Reply:
as per CSI specs we need to return available snapshots only
https://github.com/container-storage-interface/spec/blob/master/spec.md#listsnapshots